src/java.base/share/classes/java/io/RandomAccessFile.java

```diff
@@ -697,6 +697,11 @@ public void setLength(long newLength) throws IOException {
      * <p> If this file has an associated channel then the channel is closed
      * as well.
      *
+     * @apiNote
+     * When this file has an associated channel, this method may invoke
```

Comment:
"this file" should be "this stream"

Reply:
Thanks - fixed in 90a130783.

@AlanBateman, would you mind reviewing the CSR also? Thanks.